install/prepare_compute_nodes.md

```diff
@@ -131,41 +177,60 @@ This procedure needs to be done for each HPE Apollo 6500 XL645d node managed by
       After a few minutes the switch will be configured and access to the
       WebUI will be regained.
 
-1. Clear bad MAC and IP address out of kea
+   <a name="cleanup_kea"></a>
+
+3. Clear bad MAC and IP address out of KEA
+
+   **Note:** Skip this step if there was no bad MAC and IPADDR found in step 1.
 
-   Skip this step if there was no MAC and IP address found in step 1.
+   Retrieve a bearer token if you have not done so already.
+   ```
+   export TOKEN=$(curl -s -S -d grant_type=client_credentials \
+      -d client_id=admin-client \
+      -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+      https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+   ```
 
-   Example using the MAC and IP address from step 1.
+   Remove the entry from KEA that is associated with the MAC and IPADDRESS
+   gathered in section 1.1.
 
    ```
    ncn-m001:~ # curl -s -k -H "Authorization: Bearer ${TOKEN}" -X POST -H \
    "Content-Type: application/json" -d '{"command": "lease4-del", \
-   "service": [ "dhcp4" ], "arguments": {"hw-address": "94:40:c9:38:f7:b4", \
-   "ip-address": "10.254.1.38"}}' https://api-gw-service-nmn.local/apis/dhcp-kea
-
+   "service": [ "dhcp4" ], "arguments": {"hw-address": "'${MAC}'", \
+   "ip-address": "'${IPADDR}'"}}' https://api-gw-service-nmn.local/apis/dhcp-kea
+   ```
+   Expected results
+   ```
    [ { "result": 0, "text": "IPv4 lease deleted." } ]
    ```
 
-1. Clear bad ID out of HSM
+   <a name="cleanup_hsm"></a>
 
-   Skip this step if there was no ID found in step 1.
+4. Clear bad ID out of HSM
 
-   Example using the ID from step 1.
+   **Note:** Skip this step if there was no bad ID found in step 1.
 
+   Tell HSM to delete the bad ID out of the Ethernet Interfaces table
+   ```
+   ncn-m001:~ # cray hsm inventory ethernetInterfaces delete $ID
+   ```
+   Expected results
    ```
-   ncn-m001:~ # cray hsm inventory ethernetInterfaces delete 9440c938f7b4
    {
    "code": 0,
    "message": "deleted 1 entry"
    }
    ```
-
-   Everything is now configured and the node is ready to be rebooted.
+Everything is now configured and the CSM software will automatically discover
+the node after several minutes. After it has been discovered, the node is ready
+to be booted.
 
 <a name="next-topic"></a>
+
 # Next Topic
 
-   After completing the preparation for compute management nodes, the CSM product stream has
-   been fully installed and configured. Check the next topic.
+   After completing the preparation for compute nodes, the CSM product stream
+   has been fully installed and configured. Check the next topic.
 
    See [Next Topic](index.md#next_topic)
```

Comment:
See [Next Topic](index.md#next_topic) for more information on other product streams to be installed and configured after CSM.